e2e/tests/valdators.feature

```diff
@@ -7,17 +7,17 @@ Feature: Validators Namespace
     Then the user should be able to see standard output containing "Available Commands"
     And the validator should have exited without error
 
-  Scenario: User Can Stake Their Wallet
+  Scenario: User Can Stake An Address 
     Given the user has a validator
-    When the user stakes their validator with 150000000001 POKT
+    When the user stakes their validator with amount 150000000001 POKT
```

Comment:
s/POKT/uPOKT

It's 15K pot and the number here is 15 * 1e6.

Do a grep in the PR to fix elsewhere